crates/typst-cli/src/fonts.rs

```diff
@@ -1,21 +1,28 @@
-use typst::text::FontVariant;
-use typst_kit::fonts::Fonts;
-
 use crate::args::FontsCommand;
+use ecow::eco_format;
+use std::io::{self, Write};
+use typst::{diag::StrResult, text::FontVariant};
+use typst_kit::fonts::Fonts;
```

Comment:
Please respect the import order (std -> external -> crate with a blank line between each group).